hal/include/mpu_wrappers.h

```diff
@@ -0,0 +1,134 @@
+/*

```

Comment:
Was this file also generated by HALCoGen?

Reply:
It was generated by HALCoGen too

Reply:
It was generated by HALCoGen